src/test/java/ai/tecton/client/response/GetFeaturesBatchResponseTest.java

```diff
@@ -100,7 +100,7 @@ public void testMultipleSingleGetFeatures() throws Exception {
     GetFeaturesBatchResponse batchResponse =
         new GetFeaturesBatchResponse(httpResponseList, Duration.ofMillis(10), 1);
     // Verify 4 GetFeaturesResponse in the list, each with a different feature vector size
-    Assert.assertEquals(4, batchResponse.getBatchResponseList().size());
+    Assert.assertEquals(5, batchResponse.getBatchResponseList().size());
```

Comment:
How come this test changed? Didn't we add a new test for the new code path?

Reply:
This test changed cause it was reading all files in that directory. Since the number of test json files increased, the assertion number did too